### ISSUE TITLE
Cannot hit closed path area

### ISSUE THREAD

**Issue by sapics** [paperjs/paper.js#766]
Please look at http://sketch.paperjs.org/#S/ZY/NCsIwEIRfJeRShS1YjUXi0RcQPKY51HS1pWkiacRD6bubIBF/5rLfDsMuM1FTD0g5PfXoVUuBKtvE3eCDHGvfLqbKVYYEjXgd0PiRi+RECVFsSyhYKUHkYUC+YwEjBZDwky03sGYpsJIQ3fzFf9nvu/EsEenDR1ppO2LDvbvj27t0Wh+sto6TzGGTRX9e7kO9s8O6v9kuFKFcyPkJ

```javascript
new Path({
    segments:[
        [[156,146],[-146,-84],[146,84]],
        [[163,244],[146,0],   [-146,0]],
        [[156,146],[-146,84], [146,-84]]],
    closed:true,
    fillColor: 'red'
});
```

Above area cannot hit, while below area can hit.

**Comment by lehni** [comment]
You're right! Thanks for spotting. I shall investigate.

**Comment by lehni** [comment]
It turns out Curve#isLinear() returns false positives when the start and end point of the curve are in the same location. Fixing this right now.

**Comment by lehni** [comment]
Closed in af355dc82c87b55cdf4c7ddec88dec7fc0ef5f4a

**Comment by sapics** [comment]
@lehni Thank you for investigation and fixing!
